### DISCUSSION TITLE
ERROR: file:///C:/Users/16125 does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.

### DISCUSSION THREAD

**Discussion by zysqlmn020928** [open-mmlab/mmdetection#8925] [Q&A]
### Prerequisite

- [X] I have searched [the existing and past issues](https://github.com/open-mmlab/mmdetection/issues) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmdetection).

### 🐞 Describe the bug

ERROR: file:///C:/Users/16125 does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.

### Environment

Using pip 22.2.2 from D:\Anaconda\envs\torch\lib\site-packages\pip (python 3.7)
Looking in indexes: http://pypi.mirrors.ustc.edu.cn/simple

**Comment by zysqlmn020928** [comment]
![image](https://user-images.githubusercontent.com/91944653/193396748-bafa5e84-bb60-4b94-a8d4-24be1b78ba14.png)

**Comment by ZwwWayne** [comment]
This is because you are not in the mmdetection directory.
You may type `cd mmdetection` first to get into the directory of mmdet.

**Reply by zysqlmn020928** [reply]
![image](https://user-images.githubusercontent.com/91944653/193397307-af64111b-27c6-42aa-b59d-53053e4a03eb.png)

**Reply by zysqlmn020928** [reply]
我并没有这个文件诶

**Comment by 007sis** [comment]
and now?
